cordapp/contracts/src/main/java/com/newamerica/states/RequestState.java

```diff
@@ -93,14 +93,14 @@ public RequestState(String authorizedUserUsername,
 
     public String getAuthorizedUserUsername() { return authorizedUserUsername; }
```

Comment:
Maybe we should name this: _hashedUserName_ ? Because we shouldn't show their real user name for confidentiality.